resolvers.js

```diff
@@ -0,0 +1,42 @@
+// TODO: Import Mongoose here

```

Comment:
![image](https://user-images.githubusercontent.com/61930017/77873555-e9aa8880-727c-11ea-860c-b38dfdedcda6.png)
This is the output